pulp_rpm/app/modulemd.py

```diff
@@ -102,8 +104,20 @@ def create_modulemd(modulemd, snippet):
     new_module[PULP_MODULE_ATTR.ARCH] = modulemd["data"].get("arch")
     new_module[PULP_MODULE_ATTR.ARTIFACTS] = modulemd["data"].get("artifacts", {}).get("rpms", [])
     new_module[PULP_MODULE_ATTR.DESCRIPTION] = modulemd["data"].get("description")
-    new_module[PULP_MODULE_ATTR.PROFILES] = modulemd["data"].get("profiles", {})
     new_module[PULP_MODULE_ATTR.DEPENDENCIES] = modulemd["data"].get("dependencies", [])
+
+    # keep data formatted the same as it was with previous parsing implementation
+    unprocessed_profiles = modulemd["data"].get("profiles", {})
+    profiles = {}
+    if unprocessed_profiles:
+        for name, data in unprocessed_profiles.items():
+            rpms = data.get("rpms")
+            if not rpms:
+                log.warn("Got unexpected data for module: profiles failed to parse properly")
```

Comment:
does it make sense to be more specific and mention module name? otherwise this warning is not really useful